www/js/reducers/network.js

```diff
@@ -0,0 +1,59 @@
+
+import { bindAll, contains } from 'underscore'
+import Device from '../util/device'
+
+export const CONNECTION = 'pannier/network/CONNECTION';
```

Comment:
The redux Ducks standard wants an app code name as part of action strings. I chose one from a cycling dictionary.